etc/ci-coveralls-merge.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+
+# Hacked up merger for testing
+
+import json
+import os
+import os.path
+
+print("Merging coveralls json coverage") 
+merged = { 'service_name': 'travis-ci'
+         , 'service_job_id': os.environ['TRAVIS_JOB_ID']
+         , 'parallel': 'true'
```

Comment:
I guess we need to do even more here, and of course also something special for Travis... See also https://github.com/lemurheavy/coveralls-ruby/blob/master/lib/coveralls/configuration.rb and https://github.com/gap-packages/profiling/pull/65

Perhaps we could also do this instead (once the JSON header generated by profiling works as we desire, the above profiling PR is merged): Start with the `gap-coveralls.json` file, which should have the "correct" service keys; then "merge" c-coveralls.json into that. That should be easy enough, right?